quizApp.js

```diff
@@ -159,34 +160,71 @@ function displayQuestion(item, append = false) {
   if (!append) {
     section.innerHTML = "";
   }
-  const question = document.createElement("div");
-  section.appendChild(question);
-  const p = document.createElement("p");
-  p.innerText = item.question;
-  question.appendChild(p);
-  item.options.forEach((option) => {
-    const optionBtn = document.createElement("button");
-    optionBtn.classList.add("btnList");
-    optionBtn.innerText = option.text;
-    optionBtn.dataset.correct = option.isCorrect;
-    question.appendChild(optionBtn);
-    optionBtn.addEventListener("click", (event) =>
-      checkAnswer(event, question)
-    );
+  item.forEach((ques) => {
+    const question = document.createElement("div");
+    section.appendChild(question);
+
+    const p = document.createElement("p");
+    p.className = "ques";
```

Comment:
Let's get into the habit of using full words for class names. That way, it will be more descriptive and easier to read for other coders :)

Reply:
Thank you for your review. Yes, I have changed the class name to "question".